css/component.css

```diff
@@ -0,0 +1,35 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap");
+@import url("/css/style.css");
+@import url("/components/Products/products.css");
+:root {
+	--COLOR-PRIMARY: #01ccff;
+	--COLOR-SECONDARY: #27e4b5;
+	--COLOR-SOCIAL-ICON: #308ea5;
+	--PRIMARY-HOVER: #7a9ea7;
+	--NOTIFICATION-COLOR: #7cf916;
+	--offgreen: #bacfc0;
+	--darkerofseconadary: #2195d8;
```

Comment:
two suggestions:

- don't keep the color name as offgreen. What will happen is, if suppose I try to change that color to yellow, then I cannot keep the name as offgreen. So try to keep the names as logical and clear to user as possible.
- seconadary spelling is wrong. Kindly correct it.